Framework/API/inc/MantidAPI/Workspace.h

```diff
@@ -88,8 +88,8 @@ class MANTID_API_DLL Workspace : public Kernel::DataItem {
    */
   bool threadSafe() const override { return true; }
 
-  void virtual setTitle(const std::string &);
-  void setComment(const std::string &);
+  void virtual setTitle(const std::string & /*t*/);
```

Comment:
If you look in Workspace.cpp, this parameter is named `t`. It (and I believe most variable names in the PR shouldn't be commented out.